package.json

```diff
@@ -94,7 +94,7 @@
     "uuid": "^3.0.1"
   },
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/ script/ || exit 0",
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/ script/",
```

Comment:
This is required so you can run `npm run lint -- --fix`. Works better than a 🍖 to keep 🐶 from barking. Woof!